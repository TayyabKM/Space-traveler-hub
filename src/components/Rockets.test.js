import TestRenderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../redux/configureStore';
import RocketList from './Rockets';

describe('Rocket List View', () => {
  it('should render correctly', () => {
    const component = TestRenderer.create(
      <Provider store={store}>
        <RocketList />
      </Provider>,
    ).toJSON();
    expect(component).toMatchSnapshot();
  });
});
