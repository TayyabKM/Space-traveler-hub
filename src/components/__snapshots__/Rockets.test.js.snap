// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rocket List View should render correctly 1`] = `
<div
  className="container"
/>
`;
